### DISCUSSION TITLE
Getting error ENOENT: no such file or directory, stat while trying to deploy a prebuilt version on NextJS

### DISCUSSION THREAD

**Discussion by Leouss** [vercel/community#5022] [Help] [answered]
### Summary

I want to deploy my NextJS project to Vercel with the `vercel build --prod` command to create the Vercel build locally, and then deploy it to my Vercel project with the `vercel deploy --prod --prebuilt` command, however I keep getting this error:
![image](https://github.com/vercel/community/assets/64702086/e8b0991a-41d1-4547-a277-0b3ec8caee86)

However, that folder can be found in that exact path :
![image](https://github.com/vercel/community/assets/64702086/c52b5e79-06d0-4fd3-90d2-ff035e1b403d)

What am I missing? 

Thank you in advance!

### Steps to Reproduce

run 'vercel build --prod' and then run 'vercel deploy --prod --prebuilt'

**Comment by amyegan** [comment]
Hey Leo! Thanks for reaching out. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging?

**Reply by Leouss** [reply]
Hi Amy, thank you for responding. The repo on which the code project is stored is not public unfortunately, so I won't be able to share it. But what would you say are some causes that lead to this error? Also, I know that a partner working with the same project was able to deploy a prebuilt version without issues on his side. I am working with node v19.4.0, but he is working with a different node version though, 18.17.1.

**Reply by amyegan** [reply]
What happens if you try it with Node v18 or v20?

**Reply by Leouss** [reply]
Sadly, the same exact error appears trying to deploy with both v18 and v20.

Some of my recent tests regarding this problem:

1. I was testing the proposed solution from this thread https://github.com/vercel/next.js/issues/40970 but got the same error. It is a bit odd that in their case, the path was slightly different; `/vercel/path0/.vercel/output/functions/_next/data/[id]` instead of  `/vercel/path0/.vercel/output/functions/`
2. Remove .next, .vercel, node_modules before running the build command and the deployment. Same result :(

**Reply by amyegan** [reply]
Interesting. 🤔 Do you get the same error if you try to deploy a prebuilt version of a different project? A generic [create-next-app](https://nextjs.org/docs/getting-started/installation) starter would be a good test project.

I would also update to the latest version of the CLI (32.6.1) if you haven't tried that already.

**Reply by Leouss** [reply, marked as answer]
Ok, got the solution, which I believe was mentioned on that thread. If on Windows, better install Ubuntu for Windows (WSL) and run the build and deployment vercel commands using it. That solved it for me. The exact reasons why that happened are still a mystery haha, but I think some hints are mentioned on the thread too.

**Reply by amyegan** [reply]
I'm glad you got it working! Thanks for sharing your solution 😄
